api/app/weather_models/process_grib.py

```diff
@@ -148,6 +189,47 @@ def yield_data_for_stations(self, raster_band: gdal.Dataset):
 
             yield (points, values)
 
+    def yield_uv_wind_data_for_stations(self, u_raster_band: gdal.Dataset, v_raster_band: gdal.Dataset, variable: str):
```

Comment:
Is it possible to write unit tests for this function since it's new and self contained?